src/core/api.ts

```diff
@@ -78,31 +93,31 @@ export function createApi(
     return {
         subscribe,
         unsubscribe,
-        // TODO: Support assemblies (we can infer this from the spec)
         zoomTo: (viewId, position, padding = 0, duration = 1000) => {
             // Accepted input: 'chr1' or 'chr1:1-1000'
             if (!position.includes('chr')) {
                 console.warn('Genomic interval you entered is not in a correct form.');
                 return;
             }
 
+            const assembly = getTrack(viewId)?.spec.assembly;
             const chr = position.split(':')[0];
-            const chrStart = GET_CHROM_SIZES().interval?.[chr]?.[0];
+            const chrStart = GET_CHROM_SIZES(assembly).interval?.[chr]?.[0];
 
             if (!chr || typeof chrStart === undefined) {
                 console.warn('Chromosome name is not valid', chr);
                 return;
             }
 
-            const [s, e] = position.split(':')[1]?.split('-') ?? [0, GET_CHROM_SIZES().size[chr]];
+            const [s, e] = position.split(':')[1]?.split('-') ?? [0, GET_CHROM_SIZES(assembly).size[chr]];
```

Comment:
I think it would be best to write a utility function to parse this.

```typescript
function parsePosition(position: string): { name: string, start: number, end: number } {
  const [name, intervalString] = postion.split(":");
  const [start, end] = intervalString.split("-").map(s => +s.replaceAll(",", ""))
  return { name, start, end };
}
```

Reply:
Nice!